frame/aura/src/lib.rs

```diff
@@ -74,7 +74,7 @@ pub trait Trait: pallet_timestamp::Trait {
 decl_storage! {
 	trait Store for Module<T: Trait> as Aura {
 		/// The last timestamp.
-		LastTimestamp get(fn last) build(|_| 0.into()): T::Moment;
+		LastTimestamp get(fn last): T::Moment;
```

Comment:
```suggestion
		LastTimestamp get(fn last) build(|_| 0.into()): T::Moment;
```

This can be considered as a spec version change as now the genesis doesn't contains the value 0 at genesis.

it doesn't seem major but I guess better not to remove this build here.

Reply:
Makes sense, done in https://github.com/paritytech/substrate/pull/7686/commits/d782a77f1a7a2f29fe2cec6160305cd3bfea9dc1